webpack/react_app/components/TargetingHosts/__tests__/__snapshots__/TargetingHostsPage.test.js.snap

```diff
@@ -27,6 +27,7 @@ exports[`TargetingHostsPage renders 1`] = `
             "controller": "hosts",
           }
         }
+        onChange={[Function]}
```

Comment:
Another snapshot appeared 😮

Reply:
I added a searchbar mock ( instead of the foreman one) that is just `<input onchange={onchange}>` so it affected other components